packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -212,22 +221,37 @@ export default {
         }
         this.isIosBrowser = /(iPhone|iPad).*Safari/.test(navigator.userAgent);
 
-        if (this.shouldAbsolutePositionReopenLink) {
-            this.bodyObserver = new ResizeObserver(this.updateIsBodyHeightLessThanWindowHeight);
-            const bodyElement = document.documentElement || document.body;
-            this.bodyObserver.observe(bodyElement);
+        if (typeof window === 'object' && this.shouldAbsolutePositionReopenLink && !this.legacyBanner) {
+            this.createResizeObserver();
         }
     },
 
     methods: {
         /**
-         * Triggered by <body> ResizeObserver calculates if the body height less than the window
+         * Creates <body> ResizeObserver to watch for body content height changes, triggers watch.bodyHeight method
+         */
+        createResizeObserver () {
+            if (typeof ResizeObserver !== 'function') return;
+
+            const resizeCallback = entry => {
+                window.requestAnimationFrame(() => {
+                    const [{ contentRect: { height } }] = entry;
+                    this.bodyHeight = height;
+                });
+            };
+            this.bodyObserver = new ResizeObserver(resizeCallback);
+            const bodyElement = document.documentElement || document.body;
+            this.bodyObserver.observe(bodyElement);
+        },
+
+        /**
+         * Triggered vy watch.bodyHeight, calculates if body height is less than window height to absolutley position the reopen link
```

Comment:
I won't ask you to change this now, but if there are any other changes then feel free to update these at the same time

```suggestion
         * Triggered by watch.bodyHeight, calculates if body height is less than window height to absolutely position the reopen link
```